Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -161,6 +161,18 @@ open class NavigationMapView: UIView {
         }
     }
     
+    var simulation: Bool = true
+    var passiveLocationUsed: Bool = false
+    
+    /**
+     The `puckType` is used to provide  customized options for the user location puck during turn-by-turn navigation. Defaults to `UserCourseView`. Developers could switch to map SDK location layer by providing configuration to `.puck2D` or `.puck3D`.
+     */
```

Comment:
Specifics about each option should go in individual documentation comments on the `PuckType` cases.

```suggestion
    /**
     Specifies how the map displays the user’s current location, including the appearance and underlying implementation.
     
     By default, this property is set to `PuckType.default`.
     */
```